configs/KAKUTEF7HDV/config.h

```diff
@@ -31,6 +31,7 @@
 #define USE_GYRO
 #define USE_GYRO_SPI_MPU6000
 #define USE_BARO_BMP280
+#define USE_BARO_DPS310
```

Comment:
Same comment as KAKUTEF7 target
```suggestion
#define USE_BARO
#define USE_BARO_BMP280
#define USE_BARO_DPS310
```

and remove
```
#define USE_BARO
#define DEFAULT_BARO_DEVICE BARO_BMP280
```

Reply:
Also remove `DEFAULT_BARO` defines in other targets.